draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -619,11 +623,221 @@ document MAY be accompanied by a respective claim suffixed with `#integrity`, in
 
  * `vct` as defined in (#claims),
  * `extends` as defined in (#extending-type-metadata)
+ * `uri` as used in two places in (#rendering-metadata)
 
 The value MUST be an "integrity metadata" string as defined in Section 3 of
 [@!W3C.SRI]. A Consumer of the respective documents MUST verify the
 integrity of the retrieved document as defined in Section 3.3.5 of [@!W3C.SRI].
 
+
+
+# Display Metadata {#display-metadata}
+
+The `display` property is an object containing display information for the type.
+The object MUST contain a property for each language that is supported by the
+type. The property name MUST be a language tag as defined in Section 2 of
```

Comment:
I don't love the property name as a language tag value. We can discuss during a call a day later than usual and maybe an hour or two earlier but I'm wondering about an array of objects with the language tag as a property of each alongside name, description, etc.

Reply:
same thought for #claim-display-metadata

Reply:
which is also fwiw more similar to how oid4vi approached it https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-13.html#name-credential-issuer-metadata-6

Reply:
Thanks, I changed this, but used `lang` instead of `locale`. I think this might a more fitting name as this is about translations, less about the other aspects of localization.